modules/apps/frontend-data-set/frontend-data-set-views-web/types/src/main/resources/META-INF/resources/js/FDSEntries.d.ts

```diff
@@ -12,8 +12,6 @@
  * details.
  */
 
-/// <reference types="react" />
-
```

Comment:
Fake changes, Bryce is working on this. Please run `ant format-source-current-branch`, or `yarn types` on `modules` dir to get rid of these.